ddev/src/ddev/cli/meta/scripts/update_py_config.py

```diff
@@ -0,0 +1,232 @@
+# (C) Datadog, Inc. 2023-present
```

Comment:
This is just a renamed file - formerly `upgrade_python.py`.

Reply:
Any reason for not combining the 2 functions?

Reply:
I opted to make them different scripts since they target different things. The old upgrade python is just to update our hatch and pyproject config once a year or so (whenever we update the minor version). update-python-version will happen more frequently but I'm open to ideas 👍